index.html

```diff
@@ -171,5 +171,31 @@ <h3>Skills</h3>
                 </ul>
     </div>
 </section>
+<section class="form-flex">
+<p class="text-contact">I'm always interested 
+    in hearing about new projects, so if you'd 
+    like to chat please get in touch.
+</p>
+<form action="https://formspree.io/f/mnqynkkd" method="post" class="form">
+    <label for="Full name"></label>
+    <input class="inp-name" id="Full name" placeholder="Full name" type="text" maxlength="30">
+    <label for="Email address"></label>
+    <input class="inp-email" placeholder="Email address" type="email">
+    <label for="Enter text here"></label>
+    <textarea class="inp-text" placeholder="Enter text here" maxlength="500"></textarea>
```

Comment:
- [ ] While you had used great effort to implement the form inputs. I suggest that you stick to the Figma design. By implementing the following, the form inputs will look professional 💯 👍 .
   - Ensure that the color of the `text area` looks similar to the one of the Figma design. 
   - Ensure that the form inputs are centered.

- Your design 😎 
![inputs my](https://user-images.githubusercontent.com/99394374/214513987-2f5bab38-41b3-4a6d-ab36-4fc4eac6f1be.PNG)

- Figma design 😎 
![inputs](https://user-images.githubusercontent.com/99394374/214514052-1f98f8c5-2d1d-41c3-b42e-1e2732b9fdd2.PNG)